Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -835,6 +841,59 @@ extension MapboxMap {
                                  featureId: featureId,
                                  stateKey: stateKey)
     }
+
+}
+
+// MARK: - View Annotations
+
+extension MapboxMap {
+
+    // TODO: Add documentation
+    public func setViewAnnotationPositionsUpdateListenerFor(listener: ViewAnnotationPositionsListener) {
+        __map.setViewAnnotationPositionsUpdateListenerFor(listener)
+    }
+
+    /**
+     * Add view annotation.
+     *
+     * @return position for all views that need to be updated on the screen or null if views' placement remained the same.
+     */
+    public func addViewAnnotation(forId id: String, options: ViewAnnotationOptions) {
+        __map.addViewAnnotation(forIdentifier: id, options: MapboxCoreMaps.ViewAnnotationOptions(options))
+    }
+
+    /**
+     * Update view annotation if it exists.
+     *
+     * @return position for all views that need to be updated on the screen or null if views' placement remained the same.
+     */
+    public func updateViewAnnotation(forId id: String, options: ViewAnnotationOptions) {
+        __map.updateViewAnnotation(forIdentifier: id, options: MapboxCoreMaps.ViewAnnotationOptions(options))
+    }
+
+    /**
+     * Remove view annotation if it exists.
+     *
+     * @return position for all views that need to be updated on the screen or null if views' placement remained the same.
+     */
+    public func removeViewAnnotation(forId id: String) {
```

Comment:
```suggestion
    public func removeViewAnnotation(withId id: String) {
```

Reply:
This and the other functions here are named like this to match the underlying implementation. Should we rename those too? I also changed their visibility now to internal, so I wonder if we could just ignore it.

Reply:
(Same question applies to all other naming change suggestions related to these API functions.)

Reply:
The "for" in the MapboxCoreMaps method names are produced by bindgen, which isn't yet sophisticated enough to make good choices when it comes to English grammar. Don't worry about renaming those, but let's try to make our wrappers read better.

> I also changed their visibility now to internal, so I wonder if we could just ignore it.

If they're internal, it's definitely not as important to get right the first time since we can always change it later. New public APIs get a higher level of scrutiny.